includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -352,20 +356,22 @@ private function get_quiz_questions( WP_Post $quiz ) : array {
 			return [];
 		}
 
-		$quiz_questions     = [];
-		$multiple_questions = [];
+		$question_ids   = wp_list_pluck( $questions, 'ID' );
+		$quiz_questions = [];
 		foreach ( $questions as $question ) {
 			if ( 'multiple_question' === $question->post_type ) {
-				$multiple_questions[] = $question;
+				if ( Sensei()->feature_flags->is_enabled( 'block_editor_enable_category_questions' ) ) {
+					$quiz_questions[] = $this->get_category_question( $question );
+				} else {
+					foreach ( $this->get_questions_from_category( $question, $question_ids ) as $multiple_question ) {
```

Comment:
This is a change from the old behavior. It would now insert them statically in the same place as the category question.